sensing/preprocessor/gnss/gnss_poser/include/gnss_poser/convert.hpp

```diff
@@ -99,7 +101,7 @@ GNSSStat UTM2MGRS(
       std::pow(
       10, static_cast<int>(MGRSPrecision::_1_METER) -
       static_cast<int>(precision));                 // set unit as [m]
-    mgrs.z = utm.z;                                 // TODO
+    mgrs.z = utm.z;                                 // TODO(user)
```

Comment:
I have put the name of the maintainer there in my PRs. Of course the bigger problem is probably that the TODO doesn't say what is missing, but resolving that is probably out of scope for the linter PRs.